<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2520</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Агаш Пазл - Главная</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,0,0,0,0,0,0">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>50</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>50</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Программа для автоматизации производства пазлов от &quot;Агаш Пазл&quot;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDirectories">
      <property name="text">
       <string>Справочники</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPuzzles">
      <property name="text">
       <string>Пазлы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnOrders">
      <property name="text">
       <string>Заказы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnProductions">
      <property name="text">
       <string>Задания на производство товаров</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAnalyticsOrders">
      <property name="text">
       <string>Аналитика по продажам товаров</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAnalyticsProductions">
      <property name="text">
       <string>Аналитика по производству товаров</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Выход</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
